docs/releases/2.0.txt

```diff
@@ -143,6 +143,10 @@ Database backends
 * Oracle database introspection now works with auto fields, but only for newly
   created.
 
+* Added support for the :class:`~django.db.models.AutoField`,
+  :class:`~django.db.models.BigAutoField` fields as Oracle Identity Columns
```

Comment:
On Oracle, AutoField and BigAutoField are now created as identity columns. (explain the benefits of this or link "identity columns" to the Oracle docs? are there any concerns about having a mix of identity and non-identity columns in a project? should we provide a way to update existing columns to identity columns?)